lib/Parse/ParseStmt.cpp

```diff
@@ -93,6 +93,10 @@ bool Parser::isStartOfStmt() {
     Parser::BacktrackingScope backtrack(*this);
     consumeToken(tok::identifier);
     consumeToken(tok::colon);
+
+    if (Tok.is(tok::l_brace)) {
+      return true;
+    }
```

Comment:
Could you add a comment why we treat `LABEL: {` as a start of statement?

Reply:
Sure. Added.